final/MealsharingNET/Controllers/UserController.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
+using MealsharingNET.Models;
+namespace MealsharingNET.Controllers;
+
+[ApiController]
+[Route("users")]
+
+public class UserController : ControllerBase{
+  private IUserRepository _repo;
+  public UserController(IUserRepository repo){
+      _repo=repo;
+  }
+    [HttpGet("ListAllUsers")]
+    public async Task <List<User>> ListAllUser(){
```

Comment:
There is no need to have a space after Task here, `Task<List<User>>` is the return type altogether